lists/src/main/res/layout/multi_select_controls_layout.xml

```diff
@@ -19,7 +19,7 @@
         android:layout_weight="1"
         android:text="@string/select_all" />
 
-    <com.google.android.material.button.MaterialButton
+    <org.odk.collect.androidshared.ui.multiclicksafe.MultiClickSafeMaterialButton
```

Comment:
Ahh I'm not a fan of this solution. Not a long time ago we converted this button from `MultiClickSafeMaterialButton` to `MaterialButton` https://github.com/getodk/collect/pull/6098 and now we convert it back to `MultiClickSafeMaterialButton`. I won't be surprised if in a month or two we find something else in the same area that will force us to change this part of the code again.
Maybe a good solution would be to have two types of `MultiClickSafeMaterialButton`... one that would work exactly like the existing one and another one that would only block multiple clicks on exactly the same button but not other instances of it.
I'm ok with merging this temporarily if you want to fix it for v2024.2 but please file a separate issue to rework it later.

Reply:
Ah, great point! I'd forgotten that it was one of the ones we had switched over. I'll have another go.